frontend/public/components/overview/cron-job-overview.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * @deprecated [TopologySideBar] This files has been moved to @console/topology and delete this once all the side panels uses dynamic extensions
+ * @deprecated [TopologySideBar] This files has been moved to `@console/topology` and delete this once all the side panels uses dynamic extensions
```

Comment:
This fixes an ESLint `tsdoc/syntax` error where the rule thinks that `@console` is an TSDoc tag.